src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -116,24 +124,20 @@ SELECT COUNT(DISTINCT p) FROM StudentParticipation p
 
     /**
      * @param exerciseId id of exercise
-     * @param correctionRound correction round to find completed assessments by
      * @return the number of completed assessments for the specified correction round of an exam exercise
```

Comment:
This is not what the query does any more. Please explain a little bit in the javadoc what is actually in this list.

Reply:
I added a comment :)